packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -54,411 +53,41 @@ export const Default = {
-      )}
-    `;
-  }
-};
-
-export const Parts = {
-  parameters: {
-    controls: { exclude: ['base', 'header', 'summary', 'summary-icon', 'summary-border', 'content', 'content__slot'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-quickfact::part(...){outline: solid 2px red}',
-          values: ['base', 'header', 'summary', 'summary-icon', 'summary-border', 'content', 'content__slot'].map(
-            part => {
-              const outlineOffset = part === 'summary-border' ? '' : 'outline-offset: -2px';
-              return {
-                title: part,
-                // Added an outline-offset to make the outline visible for content__slot
-                value: `<style>#part-${part} sd-quickfact::part(${part}){outline: solid 2px red; ${outlineOffset};}</style><div id="part-${part}">%TEMPLATE%</div>`
-              };
-            }
-          )
-        }
-      },
-      constants: [
-        { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-        { type: 'attribute', name: 'open', value: true },
-        { type: 'attribute', name: 'expandable', value: 'true' }
-      ],
-      args
-    });
-  }
+  render: () => html`
+    <sd-quickfact expandable="" summary="Lorem Ipsum">
+      <sd-icon name="content/image" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div class="slot slot--border slot--text h-16">Default slot</div>
+    </sd-quickfact>
+  `
 };
 
 /**
- * sd-quickfacts are fully accessibile via keyboard.
+ * Use the `open` attribute to set the quickfact item to open by default.
  */
-
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">
-      ${generateTemplate({ args, constants: { type: 'attribute', name: 'expandable', value: 'true' } })}
-    </div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-quickfact');
-    await waitUntil(() => el?.shadowRoot?.querySelector('header'));
-    await userEvent.type(el!.shadowRoot!.querySelector('header')!, '{space}', { pointerEventsCheck: 0 });
-  }
+export const Open = {
+  render: () => html`
+    <sd-quickfact expandable="" open summary="Lorem Ipsum">
+      <sd-icon name="content/image" color="primary" aria-hidden="true" library="default" slot="icon"></sd-icon>
+      <div class="slot slot--border slot--text h-16">Default slot</div>
+    </sd-quickfact>
+  `
 };
 
 /**
- * This is a sample of how to group sd-quickfacts in a grid layout. Additional JavaScript is used to enable closing all other quickfacts when one is opened and to equalize the height of all summaries in a row. Open the "Show code" section to see the detailed implementation.
+ * Use the `summary` attribute to set the summary text or use the `summary` slot if you need to use custom markup.
  */
```

Comment:
For the Summary sample please use the same as in FIGMA

Reply:
Updated